crates/rpc-types-mev/src/lib.rs

```diff
@@ -12,6 +12,8 @@ pub use eth_calls::*;
 mod mev_calls;
 pub use mev_calls::*;
 
+pub mod sse;
```

Comment:
oh actually can we rename this to `mevshare`

Reply:
mevshare_sse or just mevshare?

Reply:
or maybe mevshare_events? Because the types are associated with server sent events for mevshare

Reply:
just mevshare

Reply:
done in 224edab61762bdc3566817a5a62fe77f42a93e0e